cg/models/rnafusion/rnafusion_sample.py

```diff
@@ -15,5 +15,5 @@ class RnafusionSample(NextflowSample):
     @validator("strandedness", always=True)
     def valid_value_strandedness(cls, value) -> str:
         """Verify that the strandedness value is accepted."""
-        assert value in RNAFUSION_ACCEPTED_STRANDEDNESS
+        assert value in Strandedness.accepted_values()
         return "Strandedness value not valid"
```

Comment:
We're missing the benefits of pydantic models here to perform the validation. This validator is unnecessary, you could just type hint `strandedness` and obtain the same results:

```
strandedness: Strandedness
```

Reply:
removed!